### DISCUSSION TITLE
What do those Poll XXXX options configure?

### DISCUSSION THREAD

**Discussion by alaw168** [iklein99/homebridge-smartthings#80] [Q&A] [answered]
What do Poll Locks / Garage Doors / Sensors / Switches and Lights options configure? Does it mean number of seconds / minutes between polls for each type? Or does it mean number of devices polled for each type?  There is a separate "Garage Door Max Poll" option which is required, but since I don't have a smart garage door opener, I simply ignore it.

Sorry if this is a stupid question. I tried to look for documentation / explanation but can't find any.

**Comment by iklein99** [comment, marked as answer]
It is the number of seconds to wait between each time the devices of that type are checked for status change.
